planner/core/logical_plans.go

```diff
@@ -158,10 +158,12 @@ func (p *LogicalJoin) Shallow() *LogicalJoin {
 }
 
 // GetJoinKeys extracts join keys(columns) from EqualConditions.
-func (p *LogicalJoin) GetJoinKeys() (leftKeys, rightKeys []*expression.Column) {
+func (p *LogicalJoin) GetJoinKeys() (leftKeys, rightKeys []*expression.Column, isNullEQ []bool, hasNullEQ bool) {
```

Comment:
Please add some comments about these two new arguments. 
It's better to add some specific examples about how they affect the results.

Reply:
Okay.